infrastructure/k8s/services/notify.yaml

```
@@ -0,0 +1,42 @@
+project_name: "taskcluster-notify"
+secrets:
+  aws_access_key_id: ".Values.awsAccessKeyId"
+  aws_region: ".Values.awsRegion"
+  azure_account: ".Values.tfAzureAccount"
+  denylisted_notification_table_name: "DenylistedNotification"
+  email_blacklist: "[]"
+  email_source_address: ".Values.emailSourceAddress"
+  force_ssl: "false"
+  irc_nick: ".Values.ircNick"
+  irc_password: ".Values.ircPassword"
+  irc_port: ".Values.ircPort"
+  irc_real_name: ".Values.ircRealName"
+  irc_user_name: ".Values.ircUserName"
+  monitoring_enable: "true"
+  node_env: "production"
+  publish_metadata: "false"
+  pulse_hostname: ".Values.pulseHostname"
+  pulse_password: ".Values.notifyPulsePassword"
+  pulse_username: ".Values.notifyPulseUsername"
+  pulse_vhost: ".Values.pulseVhost"
+  sqs_queue: ".Values.sqsQueue"
```

You should create this queue in the terraform (like mentioned earlier).

Add the sqs queue name to this module's outputs; https://github.com/sciurus/taskcluster/blob/cloudops/infrastructure/terraform/outputs.tf

Then you can read this from terraform like

```suggestion
  sqs_queue: ".Values.tfSqsQueue"
```